src/renderers/webgl/WebGLIndexedBufferRenderer.js

```diff
@@ -14,24 +14,10 @@ function WebGLIndexedBufferRenderer( gl, extensions, infoRender ) {
 
 	var type, size;
 
-	function setIndex( index ) {
+	function setIndex( indexProperties ) {
```

Comment:
Maybe we can rename this to `setProperties( properties )`, or even `setType( value )` and `setSize( value )`?